es/.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I didnt realize you could have a .vscode in a subfolder, how does that work? When does VS Code choose to use it?

Reply:
VS Code will only honor and use it when you open the subfolder directly.  The idea is that, if you fork/clone the repo and open it, then you can run `code es` to open the `es` subfolder which loads the Spanish linters and the page you can.

It's a common pattern on Monolithics with many projects in it